library/modules/bigip_wait.py

```diff
@@ -319,11 +335,36 @@ def _is_mprov_running_on_device(self):
             return True
         return False
 
+    def _is_vcmpd_running_on_device(self):
+        params = {
+            "command": "run",
+            "utilCmdArgs": '-c "ps aux | grep \'[v]cmpd\'"'
+        }
+        uri = "https://{0}:{1}/mgmt/tm/util/bash".format(
+            self.client.provider['server'],
+            self.client.provider['server_port']
+        )
+        resp = self.client.api.post(uri, json=params)
+        try:
+            response = resp.json()
+        except ValueError as ex:
+            raise F5ModuleError(str(ex))
+        if 'code' in response and response['code'] in [400, 403]:
+            if 'message' in response:
+                raise F5ModuleError(response['message'])
+            else:
+                raise F5ModuleError(resp.content)
+
+        if 'commandResult' in response:
+            return True
+        return False
+
 
 class ArgumentSpec(object):
     def __init__(self):
         self.supports_check_mode = True
         argument_spec = dict(
+            type=dict(default='standard', type='str'),
```

Comment:
you are missing choices here, if you have a set of options just put them in choices and supplement documentation, do not let users guess what they supposed to enter, for example albeit with just one choice for now:

https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_message_routing_peer.py#L620

Reply:
Added the choices. Good recommendation. I'll make sure to use choices from here on out.